Clarify hass availability and usage for Home Assistant Core users (#36608)
home-assistant/home-assistant.github.io: source/_docs/tools/hass.markdown

description: "Description of hass."
---

> The command-line part of Home Assistant is `hass`.\

Reviewer: ```suggestion
The command-line part of Home Assistant is `hass`.

```

> This tool is only available to Home Assistant Core users, is started from the command line on the computer running Home Assistant Core (accessed perhaps via ssh) and is not the same as Home Assistant CLI (aka hass-cli). [link](https://www.home-assistant.io/blog/2019/02/04/introducing-home-assistant-cli/)

Reviewer: Small wording adjustment.

Removed the reference of hass-cli, which is no longer part of the Home Assistant project and is a third-party tooling.

```suggestion
This tool is only available to users of the Home Assistant Core installation method. It is started from the command line on the computer running Home Assistant Core (accessed perhaps via SSH).
```

Author: In relative authority on what this thing should say, I'm so far down the list so as to not count!

Reviewer: I do not fully grasp what you mean by that response. That might be a language gap on my end. Could you clarify your wording differently?
